handler/helpers.go

```diff
@@ -0,0 +1,127 @@
+package handler
+
+import (
+	"encoding/json"
+	"fmt"
+	"html/template"
+	"net/http"
+	"net/url"
+	"reflect"
+	"strings"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/klarna/eremetic/assets"
+	"github.com/klarna/eremetic/formatter"
+	"github.com/klarna/eremetic/types"
+)
+
+// getFile handles the actual fetching of file from the agent.
+func getFile(file string, task types.EremeticTask) (int, interface{}) {
```

Comment:
not sure how that'd work with the string returns, but maybe the string returns are wrong